File: glEpisodes/src/main/java/com/kanawish/glepisodes/module/app/AppModule.java
package com.kanawish.glepisodes.module.app;

import toothpick.config.Module;

/**
 * Application level module.
 *
 *
 * GLHelper
 */
public class AppModule extends Module {

    public AppModule() {
        bind(GLHelper.class);
        // NOTE: Add bindings as needed here.
    }

}

File: glDemo/src/main/java/com/kanawish/glepisodes/di/AppModule.kt
package com.kanawish.glepisodes.di

import android.app.Application
import com.kanawish.glepisodes.module.domain.FileSystemManager
import com.kanawish.glepisodes.module.domain.ScriptManager
import com.kanawish.di.ActivityInjectionLifecycle
import toothpick.config.Module

class AppModule(appContext: Application) : Module() {

    init {
        bind(Application.ActivityLifecycleCallbacks::class.java).to(ActivityInjectionLifecycle::class.java)
        bind(ScriptManager::class.java).to(FileSystemManager::class.java)
    }
}